include/sway/desktop/fx_renderer/fx_renderer.h

```diff
@@ -32,6 +32,7 @@ struct decoration_data {
 	float dim;
 	float *dim_color;
 	bool has_titlebar;
+	bool discard_transparent;
```

Comment:
We could proabably omit this from decoration_data, since its only used by layer shell